src/main.ts

```diff
@@ -1,7 +1,8 @@
 import { platformBrowserDynamic } from '@angular/platform-browser-dynamic';
+import { enableProdMode } from '@angular/core';
 
 import { AppModule } from './app/app.module';
 
-
+enableProdMode();
```

Comment:
This enables prod mode in all environments. We only want to enable this on prod. Please use the `export const environment` from above. If `environment.production` is true then enable prod mode.